solidity/dashboard/src/sagas/coverage-pool.js

```diff
@@ -125,6 +127,10 @@ function* fetchCovPoolData(action) {
       shareOfPool
     )
 
+    const hasRiskManagerOpenAuctions = yield call(
+      Keep.coveragePoolV1.hasRiskManagerOpenAuctions
+    )
+
```

Comment:
same as above

Reply:
27e566d66d3f2927b7ed6dc7672c137ba2db746e

Reply:
fe751127495fc682637a5b9033a65f0087d9cebc - added openAuctions to the fetch cov pools data test